code/modules/wiremod/shell/brain_computer_interface.dm

```diff
@@ -495,6 +486,18 @@
 
 	open_machine()
 
+/obj/machinery/bci_implanter/proc/drop_stored_bci()
+	if (!bci_to_implant)
+		return
+	bci_to_implant.forceMove(drop_location())
+	bci_to_implant = null
+
+/obj/machinery/bci_implanter/dump_inventory_contents(list/subset)
+	// Prevents opening the machine dropping the BCI.
+	// "dump_contents()" still drops the BCI.
+	subset = contents - bci_to_implant
+	return ..()
```

Comment:
```suggestion
	return ..(contents - bci_to_implant)
```

More obvious.

Reply:
That... works? (This is copied from skillchip station code.)

Reply:
Yeah. `..()` just passes in args, and I always think that's kinda confusing.

Reply:
Guh, DM moment. Definitely will do. Should I take care of skillchip station's version too?